core/jobs/batch_jobs/certificate_assessment_attempt_cleanup_jobs_test.py

```
@@ -0,0 +1,273 @@
+            'attempt_abandoned', strict=False
+        )
+        self.assertIsNone(deleted_model)
+
+    def test_keeps_in_progress_attempt_within_deadline(self) -> None:
+        """An in-progress attempt still inside its deadline should be kept."""
+        offering_model = _create_offering_model(self, 'cert_1', 20)
+        active_attempt = _create_attempt_model(
+            self,
+            'attempt_active',
+            'cert_1',
+            datetime.datetime.utcnow() - datetime.timedelta(minutes=5),
```

Ruff (0.16.1)

[warning] `datetime.datetime.utcnow()` used

(DTZ003)